packages/fx-core/src/plugins/solution/fx-solution/v2/executeUserTask.ts

```diff
@@ -1098,7 +1098,7 @@ export async function createAuthFiles(
         await unzip(sampleZip, tabFolder);
       }
     } catch (error) {
-      // TODO: remove added code
+      await fs.remove(tabFolder);
```

Comment:
check whether folder exist before remove

Reply:
The folder is created above, it should exist
![image](https://user-images.githubusercontent.com/63089166/161369203-c4f0c1e6-f1e3-42db-9268-c20af87887f3.png)

Reply:
Actually, it's possible that OS fails to create the folder and throw an error. So, the folder doesn't necessarily exist here.

Reply:
@1yefuwang1 Error handling here improved. Thx!